src/test/java/com/jackbaretto/scrumtest/extractor/ExtractionLauncher.java

```diff
@@ -1,5 +1,7 @@
 package com.jackbaretto.scrumtest.extractor;
```

Comment:
I think this file isn't useful anymore, isn't it ? I couldn't run anything from my IDE from it.